js/src/api/util/format.js

```diff
@@ -67,3 +68,7 @@ export function toHex (str) {
 
   return `0x${(str || '').toLowerCase()}`;
 }
+
+export function isHex(str) {
```

Comment:
`util/types.js` has an isHex - it is very confusing having the 2 now, not sure what the differences are and what the various uses would be - especially from an external point. In addition, this file is for formatting functions, `util/types` would be for testing.